pynicotine/gtkgui/ui/popovers/searchfilterhelp.ui

```diff
@@ -427,8 +427,7 @@
                 <property name="visible">True</property>
                 <property name="xalign">0</property>
                 <property name="margin-start">12</property>
-                <property name="label" translatable="yes">Uses country codes defined by ISO 3166-2:
-    US will only show results from users with IP addresses in the United States. !GB will hide results that come from users in the United Kingdom (Great Britain).</property>
+                <property name="label" translatable="yes">Multiple countries can be specified with commas, spaces or | pipe seperators.</property>
```

Comment:
Perhaps we should only mention spaces everywhere in the documentation (but keep support for pipes). Less information to remember, and spaces seem like the intuitive choice.

Reply:
It does help to simply the language since there is no need to discuss the separator characters.